portfolio/src/main/webapp/script.js

```diff
@@ -79,10 +84,30 @@ function getComments() {
 
 // Creates an <li> element containing text.
 function createListElement(comment) {
-  const liElement = document.createElement('li');
-  liElement.innerHTML = comment.username + ": " + comment.content + 
-                        "<br/><i>" + comment.date + "</i>";
-  return liElement;
+  const li = document.createElement('li');
+  li.setAttribute('class', 'cmmnt');
+  const div = document.createElement('div');
+  div.setAttribute('class', 'cmmnt-content');
```

Comment:
Is there a reason not to call these "comment" and "comment-content"? Just to avoid confusion with the name of the "comments" element?